Could not find a declaration file for module '../dist/chart.esm' when using Typescript 4

## Expected Behavior
No type errors when running `tsc --noEmit`

## Current Behavior
Two errors:

```
node_modules/chart.js/auto/auto.esm.d.ts:1:23 - error TS7016: Could not find a declaration file for module '../dist/chart.esm'. '/home/eric/projects/web-www/node_modules/chart.js/dist/chart.esm.js' implicitly has an 'any' type.

1 import { Chart } from '../dist/chart.esm';
                        ~~~~~~~~~~~~~~~~~~~

node_modules/chart.js/auto/auto.esm.d.ts:3:15 - error TS7016: Could not find a declaration file for module '../dist/chart.esm'. '/home/eric/projects/web-www/node_modules/chart.js/dist/chart.esm.js' implicitly has an 'any' type.

3 export * from '../dist/chart.esm';
                ~~~~~~~~~~~~~~~~~~~


Found 2 errors.
```

## Possible Solution
Downgrading to TS 3 fixes this but other libraries we use require TS 4

## Steps to Reproduce
Use chart.js with TS 4 in any capacity.

## Context
Trying to make sure we don't have any Typescript linting errors

## Environment
Chart.js version: 3.3.2
Node.js version: 14.17.0

--- a/auto/auto.esm.d.ts
+++ b/auto/auto.esm.d.ts
@@ -1,4 +1,4 @@
-import { Chart } from '../dist/chart.esm';
+import { Chart } from '../types/index.esm';
 
-export * from '../dist/chart.esm';
+export * from '../types/index.esm';
 export default Chart;

